### DISCUSSION TITLE
How and Where to implement Text Direction

### DISCUSSION THREAD

**Discussion by efrenmarin45** [dolanmiu/docx#1048] [Q&A]
I've combed over the documentation and API for a way to implement text direction within my document and I came across TDirection and Text Direction. From what I understand it would need to be something like this: 

`new TDirection({ value: BOTTOM_TO_TOP_LEFT_TO_RIGHT })`

I also assume it would need to be placed within `children` but I keep getting an error that says it can not find BOTTOM_TO_TOP_LEFT_TO_RIGHT because it's undefined. Any insight on how I can implement text direction in my document?

**Comment by dolanmiu** [comment]
Do you mean this?

https://github.com/dolanmiu/docx/blob/master/demo/22-right-to-left-text.ts
